features/tags_and_wrangling/tag_wrangling.feature

```diff
@@ -23,225 +23,217 @@ Feature: Tag wrangling
-      And I should not see "Stargate SG-1"
-    When I follow "Wranglers"
-    Then I should see "Tag Wrangling Assignments"
-      And I should not see "Stargate SG-1"
-    When I follow "Wrangling Tools"
-    Then I should see "Fandoms by media (1)"
-    When I follow "Fandoms by media (1)"
-    Then I should see "Mass Wrangle New/Unwrangled Tags"
-      And I should not see "Stargate SG-1"
-    When I go to the new work page
-      And I select "Not Rated" from "Rating"
-      And I check "No Archive Warnings Apply"
-      And I select "English" from "Choose a language"
-      And I fill in "Fandoms" with "Stargate SG-1, Star Wars"
-      And I fill in "Work Title" with "Revenge of the Sith 2"
-      And I fill in "Characters" with "Daniel Jackson, Jack O'Neil"
-      And I fill in "Relationships" with "McShep"
-      And I fill in "content" with "That could be an amusing crossover."
-      And I press "Preview"
-      And I press "Post"
-      And The periodic tag count task is run
-      Then I should see "Work was successfully posted."
-    
-    # mass wrangling
-    When I flush the wrangling sidebar caches
-      And I follow "Tag Wrangling" within "#header"
+  Scenario Outline: Tag wrangler navigation/sidebar
+    Given the tag wrangling setup
+      And I am logged in as a tag wrangler
+    When I go to my wrangling page
     Then I should see "Wrangling Home"
-      And I should see "Wrangling Tools"
       And I should see "Fandoms by media (3)"
       And I should see "Characters by fandom (2)"
       And I should see "Relationships by fandom (1)"
-    When I follow "Fandoms by media (3)"
-    Then I should see "Mass Wrangle New/Unwrangled Tags"
-      And I should see "Stargate SG-1"
-    When I view the work "Revenge of the Sith 2"
-    Then I should see "Stargate SG-1"
+    When I follow <link_text>
+    Then I should see <heading>
     
-    # making a fandom canonical and assigning media to it
-    When I follow "Stargate SG-1"
-    Then I should see "Edit"
-    When I follow "Edit" within ".header"
+    Examples:
+      | link_text                     | heading                            |
+      | "Wranglers"                   | "Tag Wrangling Assignments"        |
+      | "Wrangling Tools"             | "Tag Wrangling"                    |
+      | "Characters by fandom (2)"    | "Mass Wrangle New/Unwrangled Tags" |
+      | "Relationships by fandom (1)" | "Mass Wrangle New/Unwrangled Tags" |
+      | "Fandoms by media (3)"        | "Mass Wrangle New/Unwrangled Tags" |
+      
+  Scenario: Edit tag page
+    Given the tag wrangling setup
+      And I am logged in as a tag wrangler
+    When I go to the "Stargate SG-1" tag page
+      And I follow "Edit" within ".header"
     Then I should see "Edit Stargate SG-1 Tag"
+      And I should see "Name"
+      And I should see "Category"
+      And I should see "Canonical"
+      And I should see "Synonym"
+      And I should see "Unwrangleable"
```

Comment:
From the issue description:
>If the tag you're editing is a fandom, the Unwrangleable section of the form should just... not be there, because fandoms are never supposed to be marked unwrangleable.

"Stargate SG-1" is a fandom tag, so I believe the "Unwrangleable" section shouldn't be visible here.

Reply:
Good catch.  I forgot to implement that part.

This test case is meant to validate the form, so I'll change this section to used a tag that's not a fandom, then add separate steps to check this behaviour for fandom tags.